utils/interfaces/schemas/library/symdb/v1/input-request.json

```diff
@@ -23,7 +23,17 @@
     },
     "symbol": {
       "type": "object",
-      "required": ["line", "name", "symbol_type", "type"],
+      "if": {
```

Comment:
I see that json schema supports comments in `"$comment"` -- please say that for Go, the debug info doesn't contain the declaration line for fields